provider/source/src/decimal/decimal_pattern.rs

```diff
@@ -55,23 +54,25 @@ impl FromStr for DecimalSubPattern {
 
         // For now, we expect one of a handful of pattern bodies.
         // TODO(#567): Generalize this to support all of UTS 35.
-        let (a, b, c, d) = match body {
-            "#,##0.###" => (3, 3, 0, 3),
-            "#,##,##0.###" => (3, 2, 0, 3),
-            "0.######" => (0, 0, 0, 6),
-            "#,##0.00" => (3, 3, 2, 2),
-            "#,#0.###" => (2, 2, 0, 3),
-            "#,##,##0.00" => (3, 2, 2, 2),
-            "#,#0.00" => (2, 2, 2, 2),
-            _ => return Err(Error::UnknownPatternBody(body.to_string())),
-        };
+        let (primary_grouping, secondary_grouping, min_fraction_digits, max_fraction_digits) =
+            match body {
+                "#,##0.###" => (3, 3, 0, 3),
+                "#,##,##0.###" => (3, 2, 0, 3),
+                "0.######" => (0, 0, 0, 6),
+                "#,##0.00" => (3, 3, 2, 2),
+                "#,#0.###" => (2, 2, 0, 3),
+                "#,##,##0.00" => (3, 2, 2, 2),
+                "#,#0.00" => (2, 2, 2, 2),
+                "#,####0.00" => (5, 2, 2, 2),
```

Comment:
Issue: If this is the pattern, it has primary and secondary grouping of 5. It looks fishy; what locale added this pattern?

Reply:
not needed after all